frameit/lib/frameit/editor.rb

```diff
@@ -338,7 +338,7 @@ def put_title_into_background(background, stack_title)
       # Resize the 2 labels if they exceed the available space either horizontally or vertically:
       image_scale_factor = 1.0 # default
       ratio_horizontal = sum_width / (image.width.to_f - horizontal_frame_padding * 2) # The fraction of the text images compared to the left and right padding
-      ratio_vertical = title.height.to_f / actual_font_size # The fraction of the actual height of the images compared to the available space
+      ratio_vertical = title.height.to_f / (title_min_height || actual_font_size) # The fraction of the actual height of the images compared to the available space
```

Comment:
Should we be using `effective_text_height` instead of `title_min_height`? 🤔

Reply:
OK. I had initially done that thinking that `effective_text_height` was not computed yet but this assumption was incorrect.